docs/user-guide.md

```diff
@@ -1,8 +1,24 @@
 # Liberty Tools for Visual Studio Code User Guide
 
+This guide provides detailed instructions on how to configure your Liberty project to use the Liberty Tools for Visual Studio Code extension.
+
 1. [Before you begin](#before-you-begin)
     - [Application requirements](#application-requirements)
     - [Settings](#settings)
+2. [Open the Liberty dashboard](#open-the-liberty-dashboard)
+3. [Run your application on Liberty using dev mode](#run-your-application-on-liberty-using-dev-mode)
+    - [Use the Liberty dashboard](#use-the-liberty-dashboard)
+    - [Use the Visual Studio Code Command Palette](#use-the-visual-studio-code-command-palette)
+    - [Start your application in dev mode](#start-your-application-in-dev-mode)
+    - [Start your application in dev mode with configuration](#start-your-application-in-dev-mode-with-configuration)
+    - [Start your application in dev mode in a container](#start-your-application-in-dev-mode-in-a-container)
+4. [Run your application's tests](#run-your-applications-tests)
+5. [View your application test reports](#view-your-application-test-reports)
+6. [Stop your application](#stop-your-application)
+7. [Debug your application](#debug-your-application)
+8. [Manually add your liberty project to the dashboard](#manually-add-your-liberty-project-to-the-dashboard)
+9. [Configure a Liberty server](#configure-a-liberty-server)
+10. [Develop with Jakarta EE and MicroProfile APIs](#develop-with-jakarta-ee-and-microprofile-apis)
 
```

Comment:
For the language in the application requirements section, mirror what we used for Intellij: 
https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/docs/user-guide.md#application-requirements